src/components/NcAppNavigationSettings/NcAppNavigationSettings.vue

```diff
@@ -3,6 +3,12 @@
   - SPDX-License-Identifier: AGPL-3.0-or-later
 -->
 
+<docs>
+⚠️ This component is deprecated and will be removed in v9.
+
+Use `<NcAppNavigationSettingsButton>` with `<NcAppSettingsButton>` instead.
```

Comment:
`NcAppSettingsButton` ? You men `NcAppSettingsDialog`?

Reply:
Yes